frontend/src/fixtures/aliasFixtures.js

```diff
@@ -0,0 +1,125 @@
+const aliasFixtures = {
+  onePropAlias: [
+    {
+      id: 1,
+      email: "yarwinliu@ucsb.edu",
+      googleSub: "4234234",
+      pictureUrl:
+        "https://lh3.googleusercontent.com/a/ACg8ocI0ocwB-nBUrhbYgpqB74jng0k8smSGTnTPh00e17a9tKJ6Ag=s96-c",
+      fullName: "Yarwin Liu",
+      givenName: "Yarwin",
+      familyName: "Liu",
+      emailVerified: true,
+      locale: null,
```

Comment:
Why do we need this? Why not just use the existing:

https://github.com/ucsb-cs156-s25/proj-dining-s25-06/blob/main/frontend/src/fixtures/usersFixtures.js

Reply:
if you need different or additional values, you can add things to the fixtures in https://github.com/ucsb-cs156-s25/proj-dining-s25-06/blob/main/frontend/src/fixtures/usersFixtures.js

But adding a whole additional file just creates more maintenance headaches for future developers.